sql/npc_list.sql

```diff
@@ -998,27 +998,27 @@ INSERT INTO `npc_list` VALUES (16834607,'csnpc','     ',0,0.000,0.000,0.000,0,50
-INSERT INTO `npc_list` VALUES (16839082,'qm_keratyrannos','???',0,-134.000,47.371,416.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839083,'qm_lentor','???',0,-248.000,47.971,403.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839084,'qm_bloodguzzler','???',0,-155.000,64.117,590.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839085,'qm_clingy_clare','???',0,150.000,17.601,90.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839086,'qm_siranpa-kamuy','???',0,370.000,1.601,10.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839087,'qm_alkonost','???',0,54.000,30.654,414.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839088,'qm_arimaspi','???',0,438.000,31.922,358.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839089,'qm_fear_gorta','???',0,630.000,33.608,410.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839090,'qm_eccentric_eve','???',0,230.413,32.278,280.677,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839091,'qm_kukulkan','???',0,-40.000,71.992,560.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839092,'qm_bloodeye_vileberry','???',0,554.000,24.198,714.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839093,'qm_eccentric_eve','???',0,215.413,32.778,280.677,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839094,'qm_kukulkan','???',0,-40.000,68.692,575.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839095,'qm_bloodeye_vileberry','???',0,539.000,24.198,714.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839096,'qm_eccentric_eve','???',0,245.413,31.778,280.677,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839097,'qm_kukulkan','???',0,-25.000,68.792,560.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
-INSERT INTO `npc_list` VALUES (16839098,'qm_bloodeye_vileberry','???',0,554.000,23.098,699.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
+INSERT INTO `npc_list` VALUES (16839078,'qm1','???',0,360.000,-16.043,-400.000,1,50,50,0,0,0,0,3,0x0000340000000000000000000000000000000000,0,'ABYSSEA',0);
```

Comment:
We've been moving away form the numbered QM pattern, what these were previously with the name telling you what it is for is preffered

Reply:
Fixed for Abyssea-La_Theine and Abyssea-Konschtat